src/lib/openpath/lib.js

```diff
@@ -6,9 +6,9 @@ import {
   getTripsForDay
 } from 'src/lib/openpath/traceRequests'
 
-import minilog from 'cozy-minilog'
+import logger from 'cozy-logger'
```

Comment:
For my personal knowledge, why is it better to change ?

Reply:
From what I heard, minilog is not suitable for services, @Merkur39 might know better

Reply:
minilog doesn't return the right log structure for cozy-stack to understand it. https://github.com/cozy/cozy-libs/tree/master/packages/cozy-logger (to see the work done by this lib)